docs/source/examples/predict-remaining-useful-life/example.ipynb

```diff
@@ -1182,7 +1182,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "y = fm.pop(lt.label_name)\n",
+    "y = fm.pop(lt.target_columns[0)\n",
```

Comment:
I think it might be a little more clear to the reader if you pop using the actual label name here instead of `target_columns[0]`, which could be confusing.